pkgs/tools/misc/wayback-machine-archiver/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, python3, fetchFromGitHub }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "wayback-machine-archiver";
+  version = "1.9.1";
+
+  # pypi doesn't include tests
+  src = fetchFromGitHub {
+    owner = "agude";
+    repo = "wayback-machine-archiver";
+    rev = "v${version}";
+    sha256 = "0dnnqx507gpj8wsx6f2ivfmha969ydayiqsvxh23p9qcixw9257x";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [ pypandoc ];
+  propagatedBuildInputs = with python3.pkgs; [ requests ];
+  checkInputs = with python3.pkgs; [ pytest pytest-runner requests-mock ];
```

Comment:
```suggestion
  checkInputs = with python3.pkgs; [ pytestCheckHook requests-mock ];
```

Reply:
1. This doesn't work

2. Upstream tests by just running pytest -vv so I did that when the hook failed.

If you have any comments that apply to this PR and isn't just general linting advice, feel free to suggest more changes.

Reply:
Then it should be patched to make work. pytest-runner does not work in the sandbox and has no use. Also pytestCheckHook does exactly what your checkPhase does.

If you paste some log I can provide more concrete help.